_posts/2024-11-04-WildFly-playing-with-generative-ai.adoc

```diff
@@ -0,0 +1,354 @@
+---
+layout: post
+title:  "Playing with Generative AI with WildFly"
+date:   2024-11-04
+tags: generative-ai rag wildfly
+author: ehugonne
```

Comment:
@ehsavoie @bstansberry I am not sure why but FYI this blog post when live on the site does not list Emmanuel as the author: https://www.wildfly.org/news/2024/11/04/WildFly-playing-with-generative-ai/

Reply:
I think https://github.com/wildfly/wildfly.org/pull/688 would solve this

Reply:
It did; thanks @tomjenkinson and @asoldano